ui/components/src/LineChart/LineChart.tsx

```diff
@@ -123,11 +123,11 @@ export const LineChart = forwardRef<ChartInstance, LineChartProps>(function Line
             // when chart undef, do not clear highlight series
             return;
           }
-          clearHighlightedSeries(chartRef.current, data.timeSeries.length);
+          clearHighlightedSeries(chartRef.current);
         },
       };
     },
-    [data.timeSeries.length]
+    []
```

Comment:
This entire component LineChart.tsx that uses 'category' xAxis should be deprecated and removed, it was replaced by TimeChart a while back in #1268